ui/lesson/ScriptingChallenge/Runner/index.tsx

```diff
@@ -40,8 +41,12 @@ export default function Runner({
   const [loading, setLoading] = useState<boolean>(true)
   const [isRunning, setIsRunning] = useState<boolean>(false)
   const [result, setResult] = useState<any | undefined>(undefined)
+  const [success, setSuccess] = useState('false')
```

Comment:
Why is this a string and not a boolean? This would prevent you from having to check `success === 'true' &&`, instead you can just do `success &&`.

Reply:
In most cases the status bar checks for a string has 4 conditions: empty, wrong, incomplete, and complete. Because we structured this challenge to only have two conditions I didnt want to redesign statusBar so i just passed true false strings